ledger/src/blockstore_processor.rs

```diff
@@ -632,6 +632,7 @@ pub struct ProcessOptions {
     /// true if after processing the contents of the blockstore at startup, we should run an accounts hash calc
     /// This is useful for debugging.
     pub run_final_accounts_hash_calc: bool,
+    pub boot_from_local_state: bool,
```

Comment:
I suggest a comment here for the next weary traveller. This is a new feature at the moment and will always be conditional on us having correctly saved local state previously on this validator. So, this bifurcated startup process and code path will always exist.

Reply:
> The validator uses these `ProcessOptions` in `core/src/validator.rs::load_blockstore()`, which calls `bank_fork_utils::load_bank_forks()`. I haven't done the plumbing here yet, but hoping it works out ok.

My question is what gets set when validator cli says to boot from local state? Shouldn't it be the same process options or shouldn't they share the same trigger mechanism?

Reply:
ah, but that option isn't even in master yet... I guess it was never merged.

Reply:
I assumed it was already in.